examples/operators/histograms/run.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict
+
+from oremda import operator
+from oremda.typing import JSONType, PortKey, RawPort, DisplayType, PlotType1D
+
+
+@operator
+def histograms(
+    inputs: Dict[PortKey, RawPort], parameters: JSONType
+) -> Dict[PortKey, RawPort]:
+    z = parameters.get("z", 0)
+    color = parameters.get("color")
+    label = parameters.get("label")
+    bins = parameters.get("bins")
+
+    data = inputs["in"].data
+
+    if data is None:
+        raise Exception("Missing input data.")
+
+    outputs = {
+        "out": RawPort(
+            meta={
+                "type": DisplayType.OneD,
+                "plot": PlotType1D.Histograms,
+                "z": z,
+                "bins": bins,
+                "color": color,
+                "label": label,
+            },
+            data=data.ravel(),
+        )
+    }
+
+    return outputs
```

Comment:
Is this a good candidate for combine with another operator? Seems like we could just get the operator this is attached to construct the right output format? The number of nodes in the current pipeline is a little intimidating!

Reply:
Yeah, I know the display operators just re-format the data a little bit. Might be nice if they could run as part of the pipeline runner rather than in a separate container, since they don't require any extra dependencies. But I'm not sure how much work that'd be.